docs/advanced/node-local-storage.md

```diff
@@ -0,0 +1,356 @@
+
+Note that using the delete policy this is different, so you may want to choose
+the policy that better fits your use case.
+
+### Automatic local persistent volume creation and handling
+
+The example manually created the `Kind: PersistentVolume` for each node. Also,
+the PV will continue to exist even if the PVC is deleted and, therefore, the
+lifecycle of the PV needs to be handled manually.
+
+Those limitation don't scale well with the number of nodes and applications,
+therefore to that end an external provisioner that will automatically create
+those objects when nodes are available and manage the lifecycle of the volume
+was created by sig-storage:
+
+	https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/
+
+### Limitations
+
+There are some limitations you might want to have in mind
+
+1. Lokomotive Kubernetes currently only exposes `/mnt/` to the kubelet, so only that
+directory can be used for node local storage. You can **manually** mount different
+devices on `/mnt/sda` or paths inside `/mnt` to your convenience.
+
+1. Using `setup_raid = "true"` will create a single RAID 0 array with all
+the spare disks and mount it in `/mnt/`. Therefore all the node storage will be
+available to only one PV, thus it will be available to only one PVC and one pod.
+This is the only automated setup supported in lokomotive at the time of this
+writting.
+
+1. You can share the volume in different PVs if you create a sub-directory per
+   PV, bind mount each of them and only exose the subdirectories. It is
+explained in detail here:
+	https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md#sharing-a-disk-filesystem-by-multiple-filesystem-pvs.
+   But take into account that it needs to be done manually.
+
+1. When using `setup_raid = "true"` the filesystem is ext4 and there is no way
+   to use another one right now.
+
+### Operation and FAQs
+
+Some recommendations and tips on how to operate are documented here:
+
+	https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md
+
+Bear in mind, however, that the provisioner is not being used in Lokomotive
+right now.
+
+There is also a useful link for FAQs:
+
+	https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/faqs.md
+
+### Useful links
+
+Here is a document with documented best practices (we are following them at the
+time of this writing, except for using UUID in fstab):
+	https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/best-practices.md
+
+And more documentation about access control that might be useful for several
+apps sharing a volume:
+	https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/#access-control
+
+And in this blog post there are some highlights not well documented anywhere
+else (will do a PR to kubernetes docs) and some useful tips from uber experience
```

Comment:
I think comment about PR is not really needed here.

>uber experience

uber? What does it mean?
```suggestion
else and some useful tips from uber experience
```

Reply:
Uber the company, sorry I wasn't clear :D

Why do you think is not needed? Because we can reference the blog post or because it does not adds no value knowing that in this document?

Reply:
>Uber the company, sorry I wasn't clear :D

Maybe `Uber's experience` then?

>Why do you think is not needed? Because we can reference the blog post or because it does not adds no value knowing that in this document?

I would create an issue instead if you would like to file a PR to Kubernetes docs, so we don't forget about it.

Reply:
Oh, yes! That was an internal note and I will either create an issue or a PR for that. Is that whan you think was not needed? I obviosly agree, that was a mistake :)